3.2/Dockerfile.fedora

```diff
@@ -1,4 +1,4 @@
-FROM fedora:25
+FROM registry.fedoraproject.org/f27/s2i-core:latest
```

Comment:
@ficap Fedora tests are now failing because F27 contains redis 4.0. Not version 3.2.

Reply:
decided to skip version check for fedora based image.
centos build failed but I think that's not my fault.

Reply:
[test]